Quotient/avatar.cpp

```diff
@@ -55,9 +55,9 @@ QImage Avatar::get(int dimension, get_callback_t callback) const
     return d->get({ dimension, dimension }, std::move(callback));
 }
 
-QImage Avatar::get(int width, int height, get_callback_t callback) const
```

Comment:
that change is questionable IMO

Reply:
I can make it the other way around, if you prefer.

Reply:
that would be nicer, IMO